service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/dataproccluster/startup.sh

```diff
@@ -932,6 +983,13 @@ if [[ -z "${INSTALLED_TERRA_VERSION}" ]]; then
   exit 1
 fi
 
+emit "--  Checking if the original Terra CLI has been renamed to terra_cli"
+
+if [[ ! -e "${TERRA_COMMAND_PATH}" ]]; then
```

Comment:
Have this check both the paths (include checking for TERRA_WRAPPER_PATH)

Reply:
Checking TERRA_WRAPPER_PATH feels redundant as it is the same as TERRA_INSTALL_PATH, but I could add it to maker it clearer to readers what's being done by the script.